people/swapna.md

```diff
@@ -0,0 +1,3 @@
+# Hi, this is Swapna
```

Comment:
Maybe you can consider deleting this file, as there are two bio files with your name.